### PR TITLE
More Ansible lint fixes

### PR DESCRIPTION
**Pull request by redhatrises**
- Add pipefail to mount templates
- Fix rpm_verify_ownership ansible remediation script as it was failing during a run

### CODE REVIEW

File: shared/templates/template_ANSIBLE_mount_option [hunk 1]
```diff
@@ -4,26 +4,35 @@
 # complexity = low
 # disruption = high
 - name: get back device associated to mountpoint
-  shell: mount | grep ' {{{ MOUNTPOINT }}} ' |cut -d ' ' -f 1
+  shell: |
+    set -o pipefail
+    mount | grep ' {{{ MOUNTPOINT }}} ' |cut -d ' ' -f 1
```
Comment (shawndwells) [inline shared/templates/template_ANSIBLE_mount_option]:
```suggestion
    mount | grep ' {{{ MOUNTPOINT }}} ' | cut -d ' ' -f 1
```

Comment (shawndwells) [reply shared/templates/template_ANSIBLE_mount_option]: nitpick about spacing

File: shared/templates/template_ANSIBLE_mount_option_var [hunk 2]
```diff
@@ -6,26 +6,35 @@
 - (xccdf-var {{{ MOUNTPOINT }}})
 
 - name: get back device associated to mountpoint
-  shell: mount | grep ' {{ {{{ MOUNTPOINT }}} }} ' |cut -d ' ' -f 1
+  shell: |
+    set -o pipefail
+    mount | grep ' {{ {{{ MOUNTPOINT }}} }} ' |cut -d ' ' -f 1
```
Comment (shawndwells) [inline shared/templates/template_ANSIBLE_mount_option_var]:
```suggestion
    mount | grep ' {{ {{{ MOUNTPOINT }}} }} ' | cut -d ' ' -f 1
```

File: shared/templates/template_ANSIBLE_mount_option (hunk 1 above)
Comment (redhatrises) [reply shared/templates/template_ANSIBLE_mount_option]: @shawndwells fixed.

File: shared/templates/template_ANSIBLE_mount_option_var (hunk 2 above)
Comment (redhatrises) [reply shared/templates/template_ANSIBLE_mount_option_var]: @shawndwells fixed.

**Pull request merged**